imports/ui/stylesheets/base/settings.less

```diff
@@ -4,6 +4,7 @@
 // Colors
 
 @color-black: #000;
+@color-grey-light-1: #E4E4E4;
```

Comment:
Why another `grey-light`? If this is really needed, can it be called something like `@color-grey-lighter`?